src/shared/utilities/childProcess.ts

```diff
@@ -171,7 +171,7 @@ export class ChildProcess {
                 this.childProcess = crossSpawn.spawn(this.command, args, options.spawnOptions)
                 this.registerLifecycleListeners(this.childProcess, errorHandler, timeout)
             } catch (err) {
-                return errorHandler(err as Error)
+                return reject(err)
```

Comment:
The exact failure was from the `spawn` syscall, exit code 8. Which is for ENOEXEC.
Couldn't find the _exact_ cause but it probably has something to do with `libuv`. Perhaps they're moving towards more of a POXIS 'spawn' rather than `execvp` semantics.